ha/util/health_generator/mock_health_event_publisher.py

```diff
@@ -205,9 +206,14 @@ def get_args() -> (argparse.Namespace, argparse.ArgumentParser):
     _conf_store = ConftStoreSearch(conf_store_req=False)
 
     data_node_ids = get_data_nodes()
+    options_regex = re.compile('(-gdt$)|(-gs$)')
 
     if len(sys.argv) > 1:
         option = sys.argv[1]
+        if not options_regex.match(option):
+            print(f"Error: Invalid Argument: {option}\n"
+                  f"Please check supported args list: ['-gdt', '-gs']")
+            sys.exit(1)
```

Comment:
args validation can be kept in get_args().   `args, parser_obj = get_args()`. 
Also, alternative args also should be considered for validation.
'-gdt', '--get-data-nodes'
'-gs', '--get-server-nodes'

Reply:
@mariyappanp in my opinion, get_args() func should be only for parsing command line arguments.
Validation should be done outside.
If you check #L211, the len of args validation is also kept outside func only, aligning to that we can extend there only.
Added
 '-gdt', '--get-data-nodes'
'-gs', '--get-server-nodes'
args as well.

Reply:
Okay. Leaving to author as my point is just a suggestion only here.